Formula/lasso.rb

```diff
@@ -21,8 +25,8 @@ class Lasso < Formula
   depends_on "openssl@1.1"
 
   resource "six" do
-    url "https://files.pythonhosted.org/packages/21/9f/b251f7f8a76dec1d6651be194dfba8fb8d7781d10ab3987190de8391d08e/six-1.14.0.tar.gz"
-    sha256 "236bdbdce46e6e6a3d61a337c0f8b763ca1e8717c03b369e87a7ec7ce1319c0a"
+    url "https://files.pythonhosted.org/packages/71/39/171f1c67cd00715f190ba0b100d606d440a28c93c7714febeca8b79af85e/six-1.16.0.tar.gz"
```

Comment:
Can we instead depend on six formula?

Reply:
![4f9](https://user-images.githubusercontent.com/3324775/125195761-1b065800-e225-11eb-96fd-9bb6e42cf82b.png)